stablehlo/tests/interpret_all_gather.mlir

```diff
@@ -1,74 +1,74 @@
 // RUN: stablehlo-translate --interpret -split-input-file %s
```

Comment:
Can we confine this to just this line instead of commenting all lines?

If there are multiple run lines we can usually do `// RUN-DISABLED:`, but since there is only one, I believe we'll get an error for there not being any RUN lines.

We could try:

```
// RUN: echo 'DISABLED' # stablehlo-translate --interpret -split-input-file %s
```